config/initializers/doorkeeper.rb

```diff
@@ -98,7 +98,7 @@
   #   http://tools.ietf.org/html/rfc6819#section-4.4.2
   #   http://tools.ietf.org/html/rfc6819#section-4.4.3
   #
-  # grant_flows %w(authorization_code client_credentials)
+  grant_flows %w(authorization_code client_credentials)
```

Comment:
I was working off this from the Doorkeeper wiki. So it should be configuring Doorkeeper to tell it what types of API authentications our app allows. We might add more down the line if necessary.
https://github.com/doorkeeper-gem/doorkeeper/wiki/Client-Credentials-flow